### PR TITLE
New method for getting attribute name

### PR DESCRIPTION
**Pull request by rpgayatri**

### CODE REVIEW

File: src/main/java/life/genny/rules/QRules.java [hunk 1]
```diff
@@ -3594,6 +3594,7 @@ public void fireAttributeChanges(QEventAttributeValueChangeMessage m) {
 			pojo.setInferred(ea.getInferred());
 			pojo.setExpired(a.getExpired());
 			pojo.setRefused(a.getRefused());
+			pojo.setChangeEvent(false);
```
Comment (DevJS) [inline src/main/java/life/genny/rules/QRules.java]: i think we can remove this?

File: src/main/java/life/genny/rules/QRules.java [hunk 2]
```diff
@@ -6473,24 +6474,24 @@ public String getUniqueCode(int numberOfDigitsForUniqueCode) {
...
-	public void processAnswers(final Answer[] answers) {
-		Arrays.stream(answers).forEach(x -> {
-			drools.insert(x);
-		});
-		drools.setFocus("AnswerProcessing");
-	}
+	// public void processAnswers(final Answer[] answers) {
+	// 	Arrays.stream(answers).forEach(x -> {
+	// 		drools.insert(x);
+	// 	});
```
Comment (DevJS) [inline src/main/java/life/genny/rules/QRules.java]: can you put this back?

File: src/main/resources/rules/rulesCurrent/shared/10_PreInit/30_PreInits/0_PreInit_ANSWER.drl [hunk 3]
```diff
@@ -33,15 +35,15 @@ rule "Pre Init Answer"
...
         rules.setState("EVENT_ANSWER"); /* other rules listening for thos state */
         rules.setState("DATA_ANSWER");
-        rules.setState("STARTED");
         rules.setState("ANSWER");
         
         if (StringUtils.isBlank(m.getAliasCode())) {
             drools.setFocus("Answer");
         } 
-        rules.processAnswers(answers);
+       /* rules.processAnswers(answers); */
```
Comment (DevJS) [inline src/main/resources/rules/rulesCurrent/shared/10_PreInit/30_PreInits/0_PreInit_ANSWER.drl]: can you put this one back too?

**Comment by rpgayatri** [comment]
All changes done..can you review again @DevJS

**Review by DevJS — APPROVED** [review]

**Pull request merged**